clap-v3-utils/src/keygen/mnemonic.rs

```diff
@@ -35,6 +35,21 @@ pub fn word_count_arg<'a>() -> Arg<'a> {
         .help(WORD_COUNT_ARG.help)
 }
 
+pub fn acquire_word_count(matches: &ArgMatches) -> usize {
+    match matches
+        .get_one::<String>(WORD_COUNT_ARG.name)
+        .unwrap()
```

Comment:
While it would be nonsensical to use this method against an `App` that did not include `WORD_COUNT_ARG`, there's nothing to stop someone doing so. Would prefer if this is at least an `expect()`, and it might be better to return a Result or Option that's handled at the call site, if the api will allow.

Reply:
Agreed. I changed the function name to `try_get_word_count` and added error handling.

Originally, I modeled the function over `acquire_language`. It seems like this comment applies to this function too, so I should probably update that function as well (maybe in a follow-up PR?).

Reply:
> Originally, I modeled the function over `acquire_language`. It seems like this comment applies to this function too, so I should probably update that function as well (maybe in a follow-up PR?).

Ah, sorry for not reading further in the file. But yes, I think `acquire_language()` would also be better if it were made fallible. Follow-up sounds great :)